dvc/api.py

```diff
@@ -13,12 +13,15 @@
 from dvc.exceptions import DvcException, NotDvcRepoError
 from dvc.external_repo import external_repo
 
+DEF_SUMMON = "Summon.yaml"
+DOBJ_SECTION = "d-objects"
```

Comment:
What does it stand for? Also, we was trying to go away from string constants) This makes it more readable.

Reply:
`dvc-objects` or `data-objects`. I'm making it `dvc-objects`.